### PR TITLE
tests: invalid JSON content

### PR DESCRIPTION
**Pull request by jirikuncar**
I have found some more corner cases. If you agree we can include it to your PR.

### CODE REVIEW

File: flask_security/core.py [hunk 1]
```diff
@@ -214,6 +214,8 @@ def _request_loader(request):
     if request.get_json(silent=True):
         if isinstance(request.json, dict):
             token = request.json.get(args_key, token)
+        else:
+            abort(422)  # we can handle only object
```
Comment (redixin) [inline flask_security/core.py]: Not sure if we really need to abort. What if library consumer want to post json list? Like ["username", "password"]? Is it possible with this library?

Comment (jirikuncar) [reply flask_security/core.py]: Then there is no way to extract token, hence 422.

Comment (redixin) [reply flask_security/core.py]: @jirikuncar 

But we have token candidate at line 213. Sorry for silly questions, I'm not familiar enough with this library yet.

Comment (jirikuncar) [reply flask_security/core.py]: I will remove this part for now together with the test update (`test_sending_auth_token_with_json`).
